devices/common_patches/h7_dbgmcu_dualcore.yaml

```diff
@@ -0,0 +1,170 @@
+# Add dualcore parts of DBGMCU periperal. From RM0399 rev 2
+
+DBGMCU:
+  CR:
+    _add:
+      DBGSTBY_D2:
+        description: Allow debug in D2 Standby mode
+        bitOffset: 5
+        bitWidth: 1
+      DBGSTOP_D2:
+        description: Allow debug in D2 Stop mode
+        bitOffset: 4
+        bitWidth: 1
+      DBGSLEEP_D2:
+        description: Allow debug in D2 Sleep mode
+        bitOffset: 3
+        bitWidth: 1
+  APB4FZ1:
+    _add:
+      IWDG2:
+        description: Independent watchdog for D2 stop when Cortex-M7 in debug mode
+        bitOffset: 19
+        bitWidth: 1
+  APB1LFZ1:
+    _add:
+      DBG_WWDG2:
```

Comment:
The other fields in this register are all named like `WWDG1` without the `DBG_` prefix, but I see that the RM says `DBG_WWDG2`. Which did you want here?

Reply:
Following the RM is the default, but useless prefixes are bad too. For minimal changes, I say strip the prefix